bugbug/repository.py

```diff
@@ -20,6 +22,14 @@
 BUG_PATTERN = re.compile('[\t ]*[Bb][Uu][Gg][\t ]*([0-9]+)')
 
 
+def hg_log(repo_dir, template=templates.changeset):
+    hg = hglib.open(repo_dir)
+    args = cmdbuilder(b'log', template=template)
```

Comment:
Hardcode the template here, removing the `template` parameter from hg_log (the parameter is not really supported anyway, given that we are parsing the result assuming the template is always the same).